CHANGES.md

```diff
@@ -87,6 +87,13 @@ To be released.
 
 ### CLI tools
 
+ -  `planet mpt diff` command became to take 4 arguments (which was 3)
+    so that it can compare state root hashes from two different
+    <abbr title="key–value">KV</abbr> stores.  The existing commands
+    like `planet mpt diff STORE A B` do not work anymore,
+    and these should be instead like `planet mpt diff STORE A STORE B`.
+    [[#1129]]
+
```

Comment:
```suggestion
 -  Store aliases used by `planet mpt` became to disallow names looking like URIs
    to disambiguate aliases from the literal store URIs.  [[#1129]]

```

Reply:
It was amended at ddecbb59c0b18dfc96c24f3c6e88c333ada57864.